### PR TITLE
Add validation 'date before or equal' and 'date after or equal'

### PR DESCRIPTION
**Pull request by febrianyuwono**

### CODE REVIEW

File: src/validators/date-after-or-equal.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      date = date.subtract(offset, unit);
+    } else {
+      message.toString = always('<%= propertyName %> must be greater than or equal to <%= ruleParams[1] %> plus <%= ruleParams[2] %> <%= ruleParams[3] %>.');
+      date = date.add(offset, unit);
+    }
+  }
+
+  date = date.subtract(1, 'day'); // Subtract 1 day to handle today
+
+  return dateInput.isAfter(date, 'day');
```
Comment (albertstevelino) [inline src/validators/date-after-or-equal.js]: there is `isSameOrAfter` in moment

File: src/validators/date-before-or-equal.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      date = date.subtract(offset, unit);
+    } else {
+      message.toString = always('<%= propertyName %> must be less than or equal to <%= ruleParams[1] %> plus <%= ruleParams[2] %> <%= ruleParams[3] %>.');
+      date = date.add(offset, unit);
+    }
+  }
+
+  date = date.add(1, 'day'); // Add 1 day to handle today
+
+  return dateInput.isBefore(date, 'day');
```
Comment (albertstevelino) [inline src/validators/date-before-or-equal.js]: `isSameOrBefore`

**Review by albertstevelino — COMMENTED** [review]
Please update and try the test case one more time

File: src/validators/date-after-or-equal.js (hunk 1 above)
Comment (febrianyuwono) [reply src/validators/date-after-or-equal.js]: updated moment for this

**Review by albertstevelino — APPROVED** [review]
test case pass! nice @febrianyuwono

**Pull request merged**